.openapi3/openapi3-openai.yaml

```diff
@@ -6187,7 +6187,7 @@ components:
           description: The model to use for image generation. Only `dall-e-2` is supported at this time.
           x-oaiTypeLabel: string
           default: dall-e-2
-        n:
+        'n':
```

Comment:
not sure what this change is about

Reply:
It's a little odd, but it shouldn't be a problem.